pkg/util/cephcmds.go

```diff
@@ -47,15 +47,14 @@ func ExecCommand(program string, args ...string) (stdout, stderr []byte, err err
 	return stdoutBuf.Bytes(), nil, nil
 }
 
-// cephStoragePoolSummary strongly typed JSON spec for osd ls pools output
-type cephStoragePoolSummary struct {
+// CephStoragePoolSummary strongly typed JSON spec for osd ls pools output
+type CephStoragePoolSummary struct {
 	Name   string `json:"poolname"`
 	Number int64  `json:"poolnum"`
 }
 
-// GetPoolID searches a list of pools in a cluster and returns the ID of the pool that matches
-// the passed in poolName parameter
-func GetPoolID(monitors string, cr *Credentials, poolName string) (int64, error) {
+// GetPools fetches a list of pools from a cluster
+func GetPools(monitors string, cr *Credentials) ([]CephStoragePoolSummary, error) {
```

Comment:
why this function is exported? is this expected to be used in other packages?

Reply:
It is currently not called form other packages, one way to possibly fix the other comment on the linter problem is to not expose this. Is that what you want?

Reply:
yes, unexporting this function should fix it.

Reply:
Fixed.